CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,7 @@
 
 - [Delta Backup Restoration] Fix assertion error [Forum #5257](https://xcp-ng.org/forum/topic/5257/problems-building-from-source/16)
 - [Delta Backup Restoration] `TypeError: this disposable has already been disposed` [Forum #5257](https://xcp-ng.org/forum/topic/5257/problems-building-from-source/20)
+- [Backups] Fix : `Error: Chaining alias is forbidden xo-vm-backups/..alias.vhd to xo-vm-backups/....alias.vhd` [Forum #5225](https://xcp-ng.org/forum/topic/5256/s3-backup-try-it)
```

Comment:
- remove space before colon
- if you can, add more relevant contextual info, like when this error occur (backuping vs restoring) or with which type(s) of remote